torch/csrc/jit/jit_opt_bisect.cpp

```diff
@@ -0,0 +1,121 @@
+#include <cstdlib>
+#include <iomanip>
+#include <sstream>
+#include <string>
+#include <vector>
+
+#include <ATen/core/function.h>
+#include <c10/util/Exception.h>
+#include <c10/util/StringUtil.h>
+#include <torch/csrc/jit/api/function_impl.h>
+#include <torch/csrc/jit/frontend/error_report.h>
+#include <torch/csrc/jit/ir/ir.h>
+#include <torch/csrc/jit/jit_opt_bisect.h>
+#include <torch/csrc/jit/serialization/python_print.h>
+
+namespace torch {
+namespace jit {
+
+static int parseOptLimit(const std::string& opt_limit) {
```

Comment:
Nit: since everywhere else we're using `int64_t` it might make sense to return it here too.